app/index.html

```diff
@@ -14,6 +14,7 @@ <h1>Título</h1>
         <ul>
             <li><a href="pages/jean.html">Professor: Jean</a></li>
             <li><a href="pages/jean_santos.html">Aluno: Jean Santos</a></li>
+            <li><a href="pages/Henrique_VieiraSacht.html">Aluno: Henrique</a></li>
```

Comment:
```suggestion
            <li><a href="pages/Henrique_VieiraSacht.html">Aluno: Henrique Vieira Sacht</a></li>
```

Nome completo seria mais apropriado